sklearn/tree/tests/test_tree.py

```diff
@@ -523,8 +523,10 @@ def test_error():
                       X, y)
         assert_raises(ValueError, TreeEstimator(max_depth=-1).fit, X, y)
         assert_raises(ValueError, TreeEstimator(max_features=42).fit, X, y)
-        assert_raises(ValueError, TreeEstimator(min_impurity_split=-1.0).fit,
-                      X, y)
+        # catch deprecation warnings for min_impurity_split
```

Comment:
I think this comment is excessive. You could put `# for min_impurity_split` at the end of the ignore_warnings line.

Reply:
hmm.. I like the idea however this might give me an error in travis build regarding line too long. How about changing it to this: 
```
# catch deprecation warnings
```
or either 
```
# catch deprecation warnings
# for min_impurity_split
```